pkg/operator/csi/csicontrollerset/csi_controller_set.go

```diff
@@ -101,11 +102,14 @@ func (c *CSIControllerSet) WithStaticResourcesController(
 func (c *CSIControllerSet) WithCredentialsRequestController(
 	name string,
 	operandNamespace string,
-	assetFunc func(string) []byte,
+	assetFunc resourceapply.AssetFunc,
 	file string,
 	dynamicClient dynamic.Interface,
 ) *CSIControllerSet {
-	manifestFile := assetFunc(file)
+	manifestFile, err := assetFunc(file)
+	if err != nil {
+		panic(fmt.Sprintf("asset: Asset(%v): %v", file, err))
```

Comment:
hrm.  I dislike this in my libraries.  it makes testing hard.  How bad is are the ripples for changing the signature to return an error?

Reply:
@jsafrane @bertinatto what do you think?